### PR TITLE
LW-9418 fixes for Conway era e2e tests

### PR DESCRIPTION
**Pull request by iccicci**
# Context

While working on LW-9418 I identified some problems which fixes deserves to be applied to `master` as well.

# Proposed Solution

Please see the commits

### CODE REVIEW

**Review by AngelCastilloB — DISMISSED** [review]
Awesome job @iccicci 🏰

File: packages/core/src/Cardano/util/computeImplicitCoin.ts [hunk 1]
```diff
@@ -150,19 +155,24 @@ const getTxDeposits = (
...
  * to be able to distinguish the deposits that are going to our rewardAccounts from the ones that could
- * potentially go to a different reward accounts that we dont control (same with reclaims).
+ * potentially go to a different reward accounts that we don't control (same with reclaims).
  */
 export const computeImplicitCoin = (
   { stakeKeyDeposit, poolDeposit }: Pick<Cardano.ProtocolParameters, 'stakeKeyDeposit' | 'poolDeposit'>,
-  { certificates, withdrawals }: Pick<HydratedTxBody, 'certificates' | 'withdrawals'>,
+  {
+    certificates,
+    proposalProcedures,
```
Comment (mirceahasegan) [inline packages/core/src/Cardano/util/computeImplicitCoin.ts:164]: Please add a unit test in `computeImplicitCoin.test.ts`.

**Review by mirceahasegan — DISMISSED** [review]
Fantastic work @iccicci 🚀

File: packages/core/src/Cardano/util/computeImplicitCoin.ts (hunk 1 above)
Comment (iccicci) [reply packages/core/src/Cardano/util/computeImplicitCoin.ts:164]: f65c3783fb should solve this

**Review by mirceahasegan — APPROVED** [review]

**Review by mkazlauskas — APPROVED** [review]
Great work! :rocket:

**Pull request merged**